diff --git a/src/main.cc b/src/main.cc
--- a/src/main.cc
+++ b/src/main.cc
@@ -124,8 +124,11 @@
 
 }  // namespace
 
-#if NODE_MAJOR_VERSION >= 10
-NAN_MODULE_WORKER_ENABLED(keytar, Init)
-#else
-NODE_MODULE(keytar, Init)
-#endif
+static void keytar_chevron_register(
+    v8::Local<v8::Object> exports,
+    v8::Local<v8::Value> module,
+    v8::Local<v8::Context> context,
+    void* priv) {
+  Init(exports);
+}
+NODE_MODULE_CONTEXT_AWARE(keytar, keytar_chevron_register)
