diff --git a/.npmignore b/.npmignore
deleted file mode 100644
index bdde9fd8f4a453a433d01506b5442923c36b4f96..0000000000000000000000000000000000000000
diff --git a/lib/natural/brill_pos_tagger/lib/Brill_POS_Tagger.js b/lib/natural/brill_pos_tagger/lib/Brill_POS_Tagger.js
index 7487f987383b3f0d4981b248bf1caf9f475bf381..9fd7b5b87eb6cd59e5c319377c329d07cbd6a2b2 100644
--- a/lib/natural/brill_pos_tagger/lib/Brill_POS_Tagger.js
+++ b/lib/natural/brill_pos_tagger/lib/Brill_POS_Tagger.js
@@ -23,7 +23,8 @@ var fs = require("fs");
 
 var TF_Parser = require('./TF_Parser');
 
-logger.setLevel('WARN');
+if (typeof logger.setLevel === 'function') logger.setLevel('WARN');
+else logger.level = 'WARN';
 
 // Tags a sentence, sentence is an array of words
 // Returns an array of tagged words; a tagged words is an array consisting of
diff --git a/lib/natural/brill_pos_tagger/lib/Predicate.js b/lib/natural/brill_pos_tagger/lib/Predicate.js
index 84874615b443ddb2db0093cebddc17d464b162d9..188039c9e86e9ca9f5db8b1cbbb9aef2de0e3cfd 100644
--- a/lib/natural/brill_pos_tagger/lib/Predicate.js
+++ b/lib/natural/brill_pos_tagger/lib/Predicate.js
@@ -18,7 +18,8 @@
 
 var log4js = require('log4js');
 var logger = log4js.getLogger();
-logger.setLevel('WARN');
+if (typeof logger.setLevel === 'function') logger.setLevel('WARN');
+else logger.level = 'WARN';
 
 var predicates = {
     // Predicates as used in the English rules in data/English/tr_from_posjs.txt
diff --git a/lib/natural/brill_pos_tagger/lib/TransformationRule.js b/lib/natural/brill_pos_tagger/lib/TransformationRule.js
index ba2363c291a616eb4118bbaf04a0c340819a8d01..1be316372d0a2996673782fde73875a657978882 100644
--- a/lib/natural/brill_pos_tagger/lib/TransformationRule.js
+++ b/lib/natural/brill_pos_tagger/lib/TransformationRule.js
@@ -21,7 +21,8 @@ var logger = log4js.getLogger();
 
 var Predicate = require("./Predicate");
 
-logger.setLevel('ERROR');
+if (typeof logger.setLevel === 'function') logger.setLevel('ERROR');
+else logger.level = 'ERROR';
 
 var category_wild_card = "*";
 
