diff --git a/lib/ignored-names.ts b/lib/ignored-names.ts
--- a/lib/ignored-names.ts
+++ b/lib/ignored-names.ts
@@ -14,9 +14,9 @@
     let left;
     this.ignoredPatterns = [];
 
-    if (Minimatch == null) { ({
-      Minimatch
-    } = require('minimatch')); }
+    if (Minimatch == null) {
+      Minimatch = require('minimatch').Minimatch;
+    }
 
     let ignoredNames = (left = chevron.config.get('core.ignoredNames')) != null ? left : [];
     if (typeof ignoredNames === 'string') { ignoredNames = [ignoredNames]; }
